new-log-viewer/src/components/StatusBar/LogLevelSelect/index.tsx

```diff
@@ -0,0 +1,273 @@
+import React, {
+    useCallback,
+    useState,
+} from "react";
+
+import {SelectValue} from "@mui/base/useSelect";
+import {
+    Box,
+    Checkbox,
+    Chip,
+    IconButton,
+    ListItemContent,
+    ListItemDecorator,
+    Option,
+    Select,
+    SelectOption,
+    Stack,
+    Tooltip,
+    TooltipProps,
+} from "@mui/joy";
+
+import AddIcon from "@mui/icons-material/Add";
+import CloseIcon from "@mui/icons-material/Close";
+import KeyboardArrowUpIcon from "@mui/icons-material/KeyboardArrowUp";
+import RemoveIcon from "@mui/icons-material/Remove";
+
+import {
+    LOG_LEVEL,
+    LOG_LEVEL_NAMES,
+    MAX_LOG_LEVEL,
+} from "../../../typings/logs";
+import {range} from "../../../utils/data";
+import LogLevelChip from "./LogLevelChip";
+
+import "./index.css";
+
+
+/**
+ * Renders a `<Tooltip/>` with its placement set to the left of the children element.
+ *
+ * @param props
+ * @param props.children
+ * @param props.rest
+ * @return
+ */
+const PlacementLeftTooltip = ({children, ...rest}: TooltipProps) => (
```

Comment:
It seems like these PlacementXxxTooltips only save one line?

Reply:
Right... Previously it was saving three lines. Let me inline it.